exit.go

```diff
@@ -0,0 +1,61 @@
+// Go support for leveled logs, analogous to https://code.google.com/p/google-glog/
+//
+// Copyright 2013 Google Inc. All Rights Reserved.
```

Comment:
Nit: Should the year be updated ?

Reply:
I can't change the year of the original copyright, but yes, adding the current one with "Kubernetes Contributors" makes sense.